device/sycl/include/traccc/sycl/clusterization/clusterization_algorithm.hpp

```diff
@@ -7,12 +7,8 @@
 
 #pragma once
 
-#include "traccc/edm/measurement.hpp"
-#include "traccc/edm/spacepoint.hpp"
-
 // clusterization
-#include "traccc/sycl/clusterization/component_connection.hpp"
-#include "traccc/sycl/clusterization/measurement_creation.hpp"
+#include "traccc/sycl/clusterization/cluster_finding.hpp"
```

Comment:
We should completely remove this class (`traccc::sycl::clusterization_algorithm`) in a future PR. Not necessarily now. But by now this algorithm is just a useless wrapper around `traccc::sycl::cluster_finding`.